include/openssl/rand.h

```diff
@@ -45,13 +45,13 @@ struct rand_meth_st {
     int (*status) (void);
 };
 
-int RAND_set_rand_method(const RAND_METHOD *meth);
-const RAND_METHOD *RAND_get_rand_method(void);
+DEPRECATEDIN_3_0(int RAND_set_rand_method(const RAND_METHOD *meth))
+DEPRECATEDIN_3_0(const RAND_METHOD *RAND_get_rand_method(void))
```

Comment:
Please read https://github.com/openssl/openssl/blob/master/doc/internal/man7/deprecation.pod

Reply:
Yes I did.  Is that the new style that was a "quiet change" because of a PR?

Reply:
Updated commit pushed that follows the new practice.